routers/api/v1/repo/release.go

```diff
@@ -58,6 +108,29 @@ func ListReleases(ctx *context.APIContext) {
 	ctx.JSON(200, rels)
 }
 
+// GetLatestRelease Gets the latest release in a repository. Draft releases and prereleases are not returned
+func GetLatestRelease(ctx *context.APIContext) {
+	// we set the pageSize to 1 to get back only one release
+	releases, err := models.GetReleasesByRepoID(ctx.Repo.Repository.ID, models.FindReleasesOptions{
+		IncludeDrafts:      false,
+		IncludePrereleases: false,
+	}, 1, 1)
+	if err != nil {
+		ctx.Error(500, "GetReleasesByRepoID", err)
+		return
+	}
+	if len(releases) <= 0 {
+		// no releases found, just return 404
+		ctx.Status(404)
+		return
+	}
+	if err := releases[0].LoadAttributes(); err != nil {
+		ctx.Error(500, "LoadAttributes", err)
+		return
+	}
+	ctx.JSON(200, releases[0].APIFormat())
```

Comment:
e we sure that `[0]` is the latest one? (I don't see anything about sorting in the PR 🙂 )